services/workflows-service/src/common/http-service/utils.ts

```diff
@@ -0,0 +1,48 @@
+import { HttpException, HttpStatus } from '@nestjs/common';
+import { AxiosError, isAxiosError } from 'axios';
+
+const _httpStatusFromAxiosMap = {
+  ENOTFOUND: HttpStatus.NOT_FOUND,
+  ECONNABORTED: HttpStatus.INTERNAL_SERVER_ERROR,
+  ERR_CANCELED: HttpStatus.INTERNAL_SERVER_ERROR, // connection canceled
+  ERR_NETWORK: HttpStatus.INTERNAL_SERVER_ERROR, // connection problems
+};
```

Comment:
Remember to add `as const` to maps. 🙂 
```suggestion
} as const;
```